app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    sum(split_integer(6, 2)) == 6
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    set(split_integer(6, 2)) == {3}
```

Comment:
The same issue here, you should use `assert` to perform the test: `assert set(split_integer(6, 2)) == {3}`.